rust/libnewsboat/src/utils.rs

```diff
@@ -271,27 +271,27 @@ mod tests {
 
     #[test]
     fn t_replace_all() {
-	assert_eq!(
+    assert_eq!(
```

Comment:
There should be two more spaces here and in all the changed lines that follow, so that asserts are indented *inside* the test.

Reply:
Now that I look at it all the asserts are formatted a little funky. I'll get those.